packages/quantic/force-app/main/default/lwc/quanticTimeframeFacet/quanticTimeframeFacet.js

```diff
@@ -268,6 +269,25 @@ export default class QuanticTimeframeFacet extends LightningElement {
     );
   }
 
+  /**
+   * Indicates whether to display the date picker.
+   */
+  get shouldDisplayDatePicker() {
+    if (!this.withDatePicker) {
+      return false;
+    }
+    if (this.dateFilterState?.range) {
```

Comment:
im not sure I understand this `this.dateFilterState?.range` could we maybe add a comment or make it a getter so it would have an easier to understand name?

Reply:
that's straight from the Headless state, no computation are being done here thus I don't see the need for the getter.
this indicates  the existence or not of a range in the date filter state.